Source/WebKit/WebProcess/GPU/media/MediaSourcePrivateRemote.cpp

```diff
@@ -162,7 +160,7 @@ void MediaSourcePrivateRemote::durationChanged(const MediaTime& duration)
     MediaSourcePrivate::durationChanged(duration);
     ensureOnDispatcher([protectedThis = Ref { *this }, this, duration] {
         auto gpuProcessConnection = m_gpuProcessConnection.get();
-        if (!isGPURunning())
+        if (!isGPURunning() || !gpuProcessConnection)
```

Comment:
We could introduce `RefPtr<Connection> validConnection()` getter since this seems to be a pattern.
It is probably harmless to call send() in case isGPURunning is false, so maybe we only need the gpuProcessConnection null check.

Reply:
calling send of isGPURunning returns false could cause an assertion on the receiving end if the message listener is gone